synapse/models/files.py

```diff
@@ -346,6 +347,26 @@ def getModelDefs(self):
                     'doc': 'The PE language id.',
                 }),
 
+                ('file:mime:macho:loadcmd', ('guid', {}), {
+                    'doc': 'A load command pulled from the Mach-O headers.',
+                }),
+
+                ('file:mime:macho:version', ('guid', {}), {
+                    'doc': 'A specific load command used to denote the version of the source used to build the Mach-O binary.',
+                }),
+
+                ('file:mime:macho:uuid', ('guid', {}), {
```

Comment:
What's our planned deconfliction strategy for these? ( mostly just so i know how to answer that if somebody asks ).  If we *really* wanted, we could potentially make them comp (file, offset) or something too, but that seems a bit hairy as well. :D

Reply:
Oh, and `guid(sha256, fileoffset)` is a 100% valid strategy :D

Reply:
I was planning to go with `guid(sha256, uuid)`, since the fileoffset aspect is a pretty malleable aspect (it's pretty easy to relocate the load commands), and the fileoffset doesn't particularly give much in the way of tell for the file.

Reply:
For the others what I had planned:
loadcmd -- guid(sha256, offset) 
* (not really a whole lot else that's unique here, so I kinda have to use the offset) 
version -- guid(sha256, versionstring)
segment -- guid(sha256, segment name, offset to segment)
* offsets are a little stronger in terms of usefulness for segments/sections, but I know there's things like overlapping segment attacks, so including the name in the guid generation helps deconflict those odd cases
section -- guid(sha256, segment name, section name, offset to section)